extension/src/test/e2e/wdio.conf.ts

```diff
@@ -47,11 +47,12 @@ export const config: Options.Testrunner = {
   capabilities: [
     {
       browserName: 'vscode',
-      browserVersion: 'insiders',
+      browserVersion: 'stable',
       'wdio:vscodeOptions': {
         extensionPath,
         userSettings: {
-          'dvc.pythonPath': getVenvBinPath(dvcDemoPath, '.env', 'python')
+          'dvc.pythonPath': getVenvBinPath(dvcDemoPath, '.env', 'python'),
+          'window.commandCenter': false
```

Comment:
This does help, but we're still getting timeouts. Decreasing the epochs to 10 also helps, but not always. The problem really seems to be on `dvc pull` on the first run.  I guess it might have to do with the progress bars or `dvcIsWorking()`. I'll look into it.